bonus_guides/G_static_assets.md

```diff
@@ -20,7 +20,7 @@ The second important file is `brunch-config.js`. This is configuration for Brunc
 
 According to this configuration Brunch will look for asset source files in `web/static`.
 
-Files and directories in `web/static/assets` will be copied to the destination without changes.
+Files and directories in `web/static/assets` will be copied to the destination without changes. Note that while all directories will be copied, only those will be served over http, which have been whitelisted in Plug.Static configuration, which by default are only the js, css, images and fonts directories.
```

Comment:
Could we reword this entire block as something like:

> All of the files in `web/static/assets` will be copied to the destination without changes.
>
> Phoenix by default will serve files in the `css`, `fonts`, `images` and `js` directories. The directories and files served by Phoenix can be set by changing the `only` option of the `Plug.Static` plug in `lib/hello_phoenix/endpoint.ex`

 > The default configuration is:

```elixir
  plug Plug.Static,
    at: "/", from: :hello_phoenix, gzip: false,
    only: ~w(css fonts images js favicon.ico robots.txt)
```